speakers.yaml

```diff
@@ -74,3 +74,39 @@ speakers:
 - id: michael
   name: Michael Duergner
   company: zalando
+- id: ewelina
+  name: Ewelina Wilkosz
+  company: Praqma
```

Comment:
Can you reference the companies from the list here as well?

Reply:
done